### PR TITLE
SB-41: NgRx features refactor

### PR DESCRIPTION
**Pull request by eccentricnode**
## Feature/Problem

The naming of the actions, and the actions strings. The inconsistency in naming throughout the NGRX reducer features, like the selectId property on initialState. The names of the Observables in the NGRX feature Facades, the naming of the selectors

## Solution

Refactored, so that there's consistency across features

## Areas of Impact

Core-State

## UI Images

N/A

## Manual Testing

- Verify CI passes

### CODE REVIEW

File: client/libs/core-state/src/lib/speakers/speakers.reducer.ts [hunk 1]
```diff
@@ -21,12 +21,13 @@ export const speakersAdapter: EntityAdapter<Speaker> = createEntityAdapter<
...
 export const initialState: SpeakersState = speakersAdapter.getInitialState({
   // set initial required properties
-  selected: null,
+  selectedSpeakerId: null,
   isLoading: false
 });
 
 const speakersReducer = createReducer(
   initialState,
+  on(SpeakersActions.selectedSpeaker),
```
[the remark this answers is not in the document]
Comment (eccentricnode) [reply client/libs/core-state/src/lib/speakers/speakers.reducer.ts]: That's a very good point.

File: client/libs/core-state/src/lib/tags/tags.effects.ts [hunk 2]
```diff
@@ -39,7 +37,7 @@ export class TagsEffects {
       ) => {
         return this.tagsService
           .create(action.tag)
-          .pipe(map((res: Tag) => TagsActions.createTagSuccess({ tag: res })));
+          .pipe(map((tag: Tag) => TagsActions.tagsCreated({ tag })));
```
Comment (VicAv99) [inline client/libs/core-state/src/lib/tags/tags.effects.ts]: tags? Should this be tag?

File: client/libs/core-state/src/lib/tags/tags.effects.ts [hunk 3]
```diff
@@ -56,7 +54,7 @@ export class TagsEffects {
       ) => {
         return this.tagsService
           .update(action.tag)
-          .pipe(map((res: Tag) => TagsActions.updateTagSuccess({ tag: res })));
+          .pipe(map((tag: Tag) => TagsActions.tagsUpdated({ tag })));
```
Comment (VicAv99) [inline client/libs/core-state/src/lib/tags/tags.effects.ts]: tags? Should this be tag?

File: client/libs/core-state/src/lib/tags/tags.effects.ts [hunk 4]
```diff
@@ -77,7 +75,7 @@ export class TagsEffects {
             switchMap((deleteConfirmed: boolean) =>
               iif(
                 () => deleteConfirmed,
-                of(TagsActions.deleteTagSuccess({ tag: action.tag })),
+                of(TagsActions.tagsDeleted({ tag: action.tag })),
```
Comment (VicAv99) [inline client/libs/core-state/src/lib/tags/tags.effects.ts]: tags? Should this be tag?

**Comment by VicAv99** [comment]
The checks are not passing, it seems not everything has been changed accordingly. Should be quick though 🚀

**Pull request merged**